### ISSUE TITLE
pm2 kill has exit code 1 on Windows

### ISSUE THREAD

**Issue by centigrade-thomas-becker** [Unitech/pm2#2484]
#### Expected behaviour

When no error occured then `pm2 kill` should exit with code 0

#### Actual behaviour

`pm2 kill` has exit code 1

#### Steps to reproduce

Running `pm2 kill` when no processes are present

```
$ pm2 kill
[PM2] Spawning PM2 daemon with pm2_home=D:\foobar\pm2
[PM2] PM2 Successfully daemonized
[PM2] Stopping PM2...
[PM2][WARN] No process found
[PM2] All processes have been stopped and deleted

$ echo %ERRORLEVEL%
1
```

Running `pm2 --kill` when some processes are present

```
$ pm2 kill
[PM2] Stopping PM2...

>>>> In-memory PM2 is out-of-date, do:
>>>> $ pm2 update
In memory PM2 version: 2.0.12
Local PM2 version: 2.0.19

[PM2] Applying action deleteProcessId on app [all](ids: 0)
[PM2] [foobar](0) ✓
[PM2] All processes have been stopped and deleted

$ echo %ERRORLEVEL%
1
```

#### Software versions used

```
OS         : Windows 8.1
node.js    : 6.8.1
PM2        : 2.0.19
```

**Comment by Unitech** [comment]
```
shant@xxx: ~/keymetrics/pm2
>>> pm2 kill
[PM2] Stopping PM2...
[PM2][WARN] No process found
[PM2] All processes have been stopped and deleted
[PM2] PM2 stopped

shant@xxx: ~/keymetrics/pm2
>>> echo $?
0
```

**Comment by vmarchaud** [comment]
We found the origin, working on a fix

**Comment by vmarchaud** [comment]
Fixed, landed in dev branch, should be published in the following days/week ;)
If you want to test it : 
```
npm install -g Unitech/pm2#development
pm2 update
```

**Comment by Unitech** [comment]
Patch available on pm2@2.1.4 (main):

```bash
$ npm install pm2 -g
$ pm2 update
```
